tfjs-automl/package.json

```diff
@@ -60,6 +60,6 @@
   "peerDependencies": {
     "@tensorflow/tfjs-backend-webgl": "^3.9.0",
     "@tensorflow/tfjs-converter": "^3.9.0",
-    "@tensorflow/tfjs-core": "^3.9.0"
```

Comment:
please update all packages to the same version. thanks

Reply:
Thanks for reviewing the PR @pyu10055 . I've updated all dependency versions to the latest release 4.19.0 . Ran the tests locally and they all passed.

Reply:
Please review again when you can.